### DISCUSSION TITLE
Hi how can i debug the application in pycharm

### DISCUSSION THREAD

**Discussion by arpit-baps00016** [ActivityWatch/activitywatch#1038] [Q&A]
When i am trying to run the main.py file from aw_server its giving me an error that 
ModuleNotFoundError: No module named 'aw_core'

i have made all the changes in pyproject.toml etc but when i do debugs its not allowing me to do that.

Can anyone please help me with this issue?

**Comment by janmechtel** [comment]
Can you give a little more background of what you are trying to do?

Are you trying to follow  the instructions to [build from source](https://docs.activitywatch.net/en/latest/installing-from-source.html) ?

You then refer to this

>The watchers all use the published versions of aw-client. If you want to use a development version of this client, you’ll need to override the definition in the pyproject.toml file via aw-client = {path = "../aw-client" }.

and you are now at the step of running `aw-server` ? Or how are you running main.py?

**Reply by arpit-baps00016** [reply]
i am trying to run aw_server's main.py file

**Reply by janmechtel** [reply]
I'm quite new to the project, just building form source myself for the first time. 

For me running `aw-qt` and `aw-server` works and there is no mention on missing references.

Are you trying to run `python3 main.py` inside `activitywatch/aw-server/aw_server` ? 

Did you clone all the subdirectories/modules?
